pipeline/e2e-tests.sh

```diff
@@ -2,7 +2,102 @@
+export KUBECONFIG=~/.kube/config
+#mkdir -p ~/.kube ~/.ssh
+ssh-keygen -R $K8S_MASTER_IP
+sshpass -p $K8S_MASTER_TKN scp -o "StrictHostKeyChecking no" root@$K8S_MASTER_IP:~/.kube/config $KUBECONFIG
+export master_ip="server: https://$K8S_MASTER_IP:6443"
+sed -i "/server: https:\\/\\/127.0.0.1/c\\ \\ \\ \\ ${master_ip}" $KUBECONFIG
+
+echo "*******"
+govc datacenter.info | grep -i path | awk '{print $2}'
+govc namespace.cluster.ls | awk -F'/' '{print $5}'
+echo "*******"
+
+echo $KUBECONFIG
+export E2E_TEST_CONF_FILE="$E2E_TEST_CONF_FILE"
+export VC_REBOOT_WAIT_TIME=1020
+export USER=root
+export VOLUME_OPS_SCALE=5
+export CLUSTER_FLAVOR="WORKLOAD"
+export STORAGE_POLICY_FOR_SHARED_DATASTORES="shared-ds-policy"
+export STORAGE_POLICY_FOR_SHARED_DATASTORES_2="shared-ds-policy-2"
+# need to create tagged vm storage policy
+export STORAGE_POLICY_FOR_NONSHARED_DATASTORES="non-shared-ds-policy"
+#export RAID_0_STORAGE_POLICY="raid-0-policy"
+export STORAGE_POLICY_WITH_THICK_PROVISIONING=thick-ds-policy-$BUILD_ID
+export SHARED_VSPHERE_DATASTORE_URL=`govc datastore.info /folder-WCP_DC/$DATACENTER/datastore/vsanDatastore | grep URL | awk '{print $2}'`
+[ -z "$SHARED_VSPHERE_DATASTORE_URL" ] && export SHARED_VSPHERE_DATASTORE_URL=`govc datastore.info /folder-WCP_DC/$DATACENTER/datastore/nfs* | grep URL | awk '{print $2}'`
+export LIST_NONSHARED_DATASTORE_URL=`govc datastore.info /folder-WCP_DC/$DATACENTER/datastore/\\*local-\\* | grep URL | awk '{print $2}'`
+[ -z "$LIST_NONSHARED_DATASTORE_URL" ] && export LIST_NONSHARED_DATASTORE_URL=`govc datastore.info /folder-WCP_DC/$DATACENTER/datastore/\\*datastore\\* | grep URL | awk '{print $2}'`
+export NONSHARED_VSPHERE_DATASTORE_URL=$(echo $LIST_NONSHARED_DATASTORE_URL | awk '{print $1;}')
+export FULL_SYNC_WAIT_TIME=350
+export SVC_NAMESPACE="e2e-test-namespace"
+export SVC_NAMESPACE_TO_DELETE="e2e-namespace-to-delete"
+export VM_NAME=$(govc ls $DATACENTER/vm |grep vmdktestvm |  awk '{ gsub("/vm/"," ");print $2}')
+export DISK_URL_PATH="https://$vcIp/folder/$VM_NAME/$VM_NAME""_1.vmdk?dcPath=$DATACENTER&dsName=vsanDatastore"
+export PATH=$PATH:`go env GOPATH`/bin
+export COMPUTE_CLUSTER_NAME="compute-cluster"
+
+export 
+export GINKGO_OPTS="--no-color --always-emit-ginkgo-writer --progress --trace -p"
+export GINKGO_FOCUS="Verify\\shealth\\stimestamp\\sannotation\\sis\\sadded\\son\\sthe\\spvc|Verify\\shealth\\sannotaiton\\sis\\snot\\sadded\\son\\sthe\\sPV|Verify\\shealth\\sannotation\\sis\\snot\\sadded\\son\\sthe\\spvc\\swhich\\sis\\spending\\sstate|Verify\\svolume\\sshrinking\\snot\\sallowed|Verify\\svolume\\sexpansion\\scan\\shappen\\smultiple\\stimes|Verify\\sstatic\\sprovisioning\\sworkflow\\son\\sSVC\\s-\\simport\\sCNS\\svolume|Verify\\sstatic\\sprovisioning\\sworkflow\\son\\sSVC\\simport\\sFCD|Verify\\sstatic\\sprovisioning\\swhen\\sAccessMode\\sis\\sReadWriteMany\\sor\\sReadOnlyMany|Specify\\sVolumeID\\sand\\sDiskURL\\stogether\\sand\\sverify\\sthe\\serror\\smessage"
```

Comment:
You may want to move this to a variable as it's much easier to modify what tests need to run in the pipeline.

Reply:
Moved to environment variable $GINKGO_FOCUS